conpot/core/loggers/mysql_log.py

```diff
@@ -60,32 +60,44 @@ def _connect(self):
 
     def _create_db(self):
         cursor = self.conn.cursor()
-        cursor.execute("""CREATE TABLE IF NOT EXISTS `events` (
-                        `id` bigint(20) NOT NULL AUTO_INCREMENT,
-                        `sensorid` text NOT NULL,
-                        `session` text NOT NULL,
-                        `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-                        `remote` text NOT NULL,
-                        `protocol` text NOT NULL,
-                        `request` text NOT NULL,
-                        `response` text NOT NULL,
-                        PRIMARY KEY (`id`)
-                        ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
-                       """)
+        cursor.execute(""" SELECT count(*) FROM information_schema.tables WHERE table_name = %s and table_schema=%s""",("events",self.db)) 
+        if (cursor.fetchone()[0]) == 0:
+            cursor.execute("""CREATE TABLE IF NOT EXISTS `events` (
```

Comment:
Yes, that is true, i didn't look carefully into the SQL query. However, the problem persists in my deployment without the code above. 
I made a step by step debug and figure out that:
the _create_db() run nicely when I start up conpot, however, when I start to communicate to conpot ( e.g. Modbus), The log() method will throw exception as I described in #302. The exception of log() will call _connect() which in turn call _create_db() for the second time. This is where conpot hangs with error: 
" ERROR:root:Stopping because <Greenlet at 0x7f0c14596370: <bound method LogWorker.start of <conpot.core.loggers.log_worker.LogWorker object at 0x7f0c14643690>>> died: "
To sum up, conpot hangs when _create_db() is called for the second time ( I don't know why because I have tried to test to run _create_db() continuously to mysql server, but there was no problem happened). However, I think it is better to check whether table "events" exists than relying on "IF NOT EXISTS", and an additional check in log() would help to avoid unnecessary warnings.